docs/0.2-DRAFT/index.md

```diff
@@ -1394,7 +1408,7 @@ If [thumbnail](http://schema.org/thumbnail)s are incidental to the data set, the
 
 (These are notes for now)
 
-DataCite can accept metadata in Schema.org format if it is framed correctly via the Bolognese crosswalk to. 
+DataCite can accept metadata in Schema.org format if it is framed correctly, via the Bolognese crosswalk.
 
```

Comment:
Discussed this & decided to remove DataCite/Bolognese reference in favour of just saying that, when displaying an RO-Crate website, a textual citation should be displayed (possibly with reference to [Force11 Data Citation Principles](https://www.force11.org/datacitationprinciples))